http/src/request/guild/create_guild.rs

```diff
@@ -67,6 +71,104 @@ struct CreateGuildFields {
     verification_level: Option<VerificationLevel>,
 }
 
+/// Fillable struct to send new roles to a guild create.
+#[derive(Clone, Debug, Default)]
+pub struct RoleFragment {
```

Comment:
Fragment is an uncommon word so if anyone has any ideas that isn't "partial",

Reply:
`Incomplete`?